Update final_report.md

se-mpervivum opened the pull request:
Added more robust documentation for Python-based segmentation

se-mpervivum committed:
Update final_report.md

Added more robust documentation for Python-based segmentation

cmerikson reviewed (changes requested):
Can you change the references to "threshold" to "Python threshold"? There is already a threshold approach, so this needs to be distinguished.

se-mpervivum committed:
Update final_report.md

Made the necessary clarifications/adjustments about Python vs. Julia threshold segmentation
